test/cypress/integration/22-collective.edit.test.js

```diff
@@ -59,12 +59,11 @@ describe('edit collective', () => {
     cy.getByDataCy('create-collective-mini-form').should('not.exist'); // Wait for form to be submitted
     cy.getByDataCy('confirmation-modal-continue').click();
     cy.get('[data-cy="member-1"] [data-cy="member-pending-tag"]').should('exist');
-    cy.getInbox().should('have.length', 2);
+    cy.getEmail(({ subject }) => subject.includes('Invitation to join CollectiveToEdit'));
 
     // Re-send the invitation email
+    cy.clearInbox();
     cy.getByDataCy('resend-invite-btn').should('exist').first().click({ force: true });
-    cy.wait(200); // Wait for email
-    cy.getInbox().should('have.length', 3);
 
```

Comment:
We're dropping a feature here: the test was making sure that the re-send invite button was actually re-sending another email. Can we find a way to fix the test without decreasing the coverage?

Reply:
I think it still does that, right? We clear the inbox before we click the re-send invite button, so righ after when we attempt to open an email with the invitation if the email was not re-send the assertion fails.

Reply:
You're right! All good for me then